package.json

```diff
@@ -56,19 +56,25 @@
     "tslib": "^2.3.0",
     "typescript": "~4.6.2",
     "uuid": "^8.3.2",
-    "yargs": "^17.0.0",
-    "zone.js": "^0.11.4"
+    "yargs": "^17.0.0"
   },
   "devDependencies": {
     "@actions/core": "^1.4.0",
     "@actions/github": "^5.0.0",
     "@angular/animations": "14.0.0-next.9",
     "@angular/bazel": "14.0.0-next.9",
+    "@angular/cdk": "14.0.0-next.7",
+    "@angular/cli": "~13.3.0",
```

Comment:
Shouldn't we use the v14 branch here as well? Similar to other Angular deps, or to the CLI devkit